internal/pkg/skuba/kubernetes/versions.go

```diff
@@ -33,6 +33,7 @@ type Component string
 
 const (
 	Cilium        Addon = "cilium"
+	CNI           Addon = "cni"
```

Comment:
I assume cilium is one of the CNI, so having two Cilium and CNI both types is Addon seems a bit weird to me.

Reply:
Oops, that got left around as an artifact from some experimentation earlier. Good catch.